### ISSUE TITLE
[Minor] After "--revert" option, the "tcpdump" package shows unmet dependencies

### ISSUE THREAD

**Issue by Bitpalast** [plesk/centos2alma#212]
**Describe the bug**
After an issue during the 1st phase of conversion I used the "--revert" option to revert to the state before starting the conversion. I accidentally found that the optional package tcpdump showed missing dependencies afterwards:

```
# package-cleanup --problems
Loaded plugins: fastestmirror
Package 14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap >= ('14', '1.5.3', '10')
Package 14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap.so.1()(64bit)

```
As that package was not needed, I removed it by "yum remove tcpdump". Afterwards "package-cleanup --problems" showed no more issues.

**Comment by Bitpalast** [comment]
On further tests this could not be reproduced :-(

**Comment by SandakovMM** [comment]
Hm, unable to reproduce as well

**Comment by Bitpalast** [comment]
I was now able to reproduce it:

1) # ./centos2alma --start --no-reboot
2) Repeat the 1st phase: # ./centos2alma --start -no-reboot
3) Revert: # ./centos2alma --revert

Now on further yum operations we see:
** Found 3 pre-existing rpmdb problem(s), 'yum check' output follows:
plesk-core-18.0-2.centos.7+p18.0.59.2+t240229.1010.x86_64 has missing requires of psa-phpmyadmin >= ('0', '5.2.1', None)
14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap >= ('14', '1.5.3', '10')
14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap.so.1()(64bit)

And
# package-cleanup --problems
Loaded plugins: fastestmirror
Package 14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap >= ('14', '1.5.3', '10')
Package 14:tcpdump-4.9.2-4.el7_7.1.x86_64 has missing requires of libpcap.so.1()(64bit)

**Comment by SandakovMM** [comment]
Should be fixed in scope of #218.
Please find attached archive with new version of the tool - [centos2alma.zip](https://github.com/plesk/centos2alma/files/14758587/centos2alma.zip)
